logging/efk-logging-elasticsearch.adoc

```diff
@@ -10,6 +10,11 @@ toc::[]
 You can add and remove nodes, configure storage for your Elasticsearch cluster, and 
 define how shards are replicated across data nodes in the cluster, from full replication to no replication.
 
+Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and CPU limits 
+unless you speccify otherwise the ClusterLogging custom resource. The initial set of {product-title} nodes might not be large enough 
+to support the Elasticsearch cluster. You must add additional nodes to the  {product-title} cluster to run with the recommended
```

Comment:
@jcantrill The user needs to add OpenShift nodes or ES?

Reply:
User would need to provide openshift nodes that have the capacity to support the memory requirements of ES